src/main/scala/uk/gov/hmrc/play/audit/http/HttpAuditing.scala

```diff
@@ -25,16 +28,23 @@ import uk.gov.hmrc.http.hooks.HttpHook
 import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
 import uk.gov.hmrc.time.DateTimeUtils
 
+import scala.xml._
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.matching.Regex
 
 trait HttpAuditing extends DateTimeUtils {
 
   val outboundCallAuditType: String = "OutboundCall"
 
+  private val MaskValue = "########"
+
   def auditConnector: AuditConnector
   def appName: String
   def auditDisabledForPattern: Regex = """http(s)?:\/\/.*\.(service|mdtp)($|[:\/])""".r
+  def maskField(field:String) = {
+    val lower = field.toLowerCase()
+    lower.contains("password") || lower.contains("passwd")
```

Comment:
Consider having Set(lowercase sensitive terms).contains(field.toLowerCase). And this isn't extendable from previous experience with play upgrade?